### PR TITLE
Minor improvements to allocd

### PR DESCRIPTION
**Pull request by jemoreira**

### CODE REVIEW

File: cvd/allocd/resource_manager.cpp [hunk 1]
```diff
@@ -33,9 +33,9 @@
 #include <string>
 
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/alloc_utils.h"
-#include "host/libs/allocd/request.h"
-#include "host/libs/allocd/utils.h"
+#include "alloc_utils.h"
+#include "request.h"
+#include "utils.h"
```
Comment (Databean) [inline cvd/allocd/resource_manager.cpp:38]: nit: should these now be `allocd/`?

Should also be reordered.

File: cvd/allocd/resource_manager.h [hunk 2]
```diff
@@ -26,10 +26,10 @@
 #include <set>
 
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/alloc_utils.h"
-#include "host/libs/allocd/request.h"
-#include "host/libs/allocd/resource.h"
-#include "host/libs/allocd/utils.h"
+#include "alloc_utils.h"
```
Comment (Databean) [inline cvd/allocd/resource_manager.h:29]: nit: should these now be `allocd/`?

Should also be reordered.

File: cvd/allocd/allocd.cpp [hunk 3]
```diff
@@ -34,9 +34,9 @@
 #include <thread>
 
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/alloc_utils.h"
-#include "host/libs/allocd/request.h"
-#include "host/libs/allocd/resource_manager.h"
+#include "alloc_utils.h"
+#include "request.h"
+#include "resource_manager.h"
```
Comment (Databean) [inline cvd/allocd/allocd.cpp:39]: nit: should these now be under `allocd/`?

Should also be reordered.

File: cvd/allocd/test_client.cpp [hunk 4]
```diff
@@ -22,8 +22,8 @@
 #include <iostream>
 
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/request.h"
-#include "host/libs/allocd/utils.h"
+#include "request.h"
+#include "utils.h"
```
Comment (Databean) [inline cvd/allocd/test_client.cpp:26]: nit: should these now be `allocd/`?

Should also be reordered.

File: cvd/allocd/utils.cpp [hunk 5]
```diff
@@ -14,14 +14,14 @@
...
-#include "host/libs/allocd/utils.h"
+#include "utils.h"
 
 #include <cstdint>
 #include <optional>
 
 #include "common/libs/fs/shared_buf.h"
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/request.h"
+#include "request.h"
```
Comment (Databean) [inline cvd/allocd/utils.cpp:24]: nit: should these now be `allocd/`?

File: cvd/allocd/utils.h [hunk 6]
```diff
@@ -23,7 +23,7 @@
 #include <string>
 
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/request.h"
+#include "request.h"
```
Comment (Databean) [inline cvd/allocd/utils.h:26]: nit: should this now be `allocd/`?

Should also be reordered.

File: cvd/allocd/resource.cpp [hunk 7]
```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-#include "host/libs/allocd/resource.h"
+#include "resource.h"
 
 #include <android-base/logging.h>
 
-#include "host/libs/allocd/alloc_utils.h"
+#include "alloc_utils.h"
```
Comment (Databean) [inline cvd/allocd/resource.cpp:21]: nit: should these now be `allocd/`?

File: cvd/allocd/alloc_utils.cpp [hunk 8]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include "host/libs/allocd/alloc_utils.h"
+#include "alloc_utils.h"
```
Comment (Databean) [inline cvd/allocd/alloc_utils.cpp:16]: nit: should this now be `allocd/`?

File: cvd/allocd/alloc_utils.h [hunk 9]
```diff
@@ -25,7 +25,7 @@
 #include <sstream>
 
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/request.h"
+#include "request.h"
```
Comment (Databean) [inline cvd/allocd/alloc_utils.h:28]: nit: should these now be `allocd/`?

**Review by Databean — APPROVED** [review]

File: cvd/allocd/alloc_utils.cpp (hunk 8 above)
Comment (jemoreira) [reply cvd/allocd/alloc_utils.cpp:16]: The stuff under `cuttlefish/host/libs/<module>/<header>` are included as `"host/libs/<module>/<header>` so I thought the most symmetric way to do it here is to exclude the top level `allocd` directory. I don't have a preference either way though.

File: cvd/allocd/allocd.cpp (hunk 3 above)
Comment (jemoreira) [reply cvd/allocd/allocd.cpp:40]: reorders done

**Pull request merged**